pkg/omf/init.fish

```diff
@@ -0,0 +1,3 @@
+function init -a path --on-event init_omf
```

Comment:
Yeah me neither... this kinda defeats the purpose of having the `init_omf` event. BUT, I couldn't think of a better and easier to use idea yet...